tests/infrastructure/golden_images/test_common_templates_data_volumes.py

```diff
@@ -167,8 +167,8 @@ def test_vm_with_existing_dv(data_volume_scope_function, vm_from_template_with_e
 )
 @pytest.mark.polarion("CNV-5529")
 def test_vm_dv_with_different_sc(
```

Comment:
So I doubt we will run on a cluster with NFS these days... HPP+OCS is much more likely to happen, but still, not all clusters will have them both. And our tests need to be storage agnostic, so hard coding the names doesn't look like a good idea. What exactly do we want to test here?

Reply:
We don't deploy a cluster with nfs anymore. So may be mark this test so that it is collected when a cluster has nfs or consider if it can be run on cluster with ocs + hpp? if the goal is to test that vms can be cloned with different sc, then may be it is valid for ocs+hpp?

Reply:
okay i will test with other combinations.

Reply:
I don't like the approach for hard-coding the storage classes names. Soon we may (and most likely will) stop using these storages and will have something new - this test will not be able to run. So we need to understand what we are testing - cross-provisioner cloning? FS to Block cross-provisioner cloning?

Reply:
@jpeimer makes sense to what you are saying but for now i will keep it out of the scope of this PR. I will handle in a different one.I'll have a jira and plan it

Reply:
Sure, I agree it's out of scope of this PR

Reply:
https://issues.redhat.com/browse/CNV-57583
@jpeimer ^^